src/client/visualizations/table/table.tsx

```diff
@@ -228,13 +229,101 @@ export class Table extends BaseVisualization<TableState> {
     }
   }
 
-  renderInternals() {
-    var { clicker, essence, stage, openRawDataModal } = this.props;
-    var { datasetLoad, flatData, scrollLeft, scrollTop, hoverMeasure, hoverRow } = this.state;
-    var { splits, dataSource } = essence;
-    var splitDimension = splits.get(0).getDimension(dataSource.dimensions);
+  onSimpleTableSizeChange(dimensions: {bodyWidth: number}) {
```

Comment:
There is no need for this function or for the `bodyWidth` variable because `bodyWidth === stage.width - SPACE_LEFT - SEGMENT_WIDTH - SPACE_RIGHT` where `const { stage } = this.props`